app/src/main/java/fr/groggy/racecontrol/tv/db/RaceControlTvDatabase.kt

```diff
@@ -18,7 +18,8 @@ import fr.groggy.racecontrol.tv.db.session.SessionEntity
         SeasonEntity::class,
         SessionEntity::class
     ],
-    version = 5
+    version = 5,
+    exportSchema = true
```

Comment:
I believe this exportSchema has no effect if we do not define the location of the schema export.

I'd recommend that you remove it